src/styles/layout/_component-grid.scss

```diff
@@ -19,7 +19,7 @@ $breakpoint-accordion-medium-550: 34rem;
   &__heading {
     position: relative;
     font-family: Viga, sans-serif;
-    font-size: 1.5625rem; // 25px/16
+    font-size: map.get($font-sizes, "mobile-heading"); // 28px/16
```

Comment:
Maybe drop off the px comment on this one too? Otherwise LGTM 👏 Ill leave it to the other reviewers on this PR!

Reply:
Thanks for catching that! 

https://github.com/sparkbox/accessible-components/pull/118/commits/6b82dfcb97c18c7002eeb924eee95b3bc91d0ec8